frontend/src/components/Reader/Vocabulary.js

```diff
@@ -8,26 +8,28 @@ import { Card, Dimmer, Loader } from "semantic-ui-react";
 import Word from "./Word";
 
 const GET_WORDS_IN_TEXT = gql`
-  query getWordsInText($text: String!) {
-    wordsInText(text: $text) {
+  query getWordsInText($language: String!, $text: String!) {
+    wordsInText(language: $language, text: $text) {
       language
       text
       partOfSpeech
       lemma
-      definitions
-      ... on ChineseWord {
-        hsk
-        pinyin
-      }
     }
   }
 `;
 
+// Removed
+// definitions
+// ... on ChineseWord {
+//   hsk
+//   pinyin
+// }
+
 const Vocabulary = props => {
-  const { text, submitted } = props;
+  const { language, text, submitted } = props;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Delete `;`](https://app.codacy.com/manual/lucaskjaero/foreign-language-reader/pullRequest?prid=5099894)

Reply:
This is the opposite of the ESlint rules.